libraries/api-extractor/src/aedoc/ApiDocumentation.ts

```diff
@@ -311,13 +311,16 @@ export default class ApiDocumentation {
         }
       } else if (token.type === TokenType.Text)  {
         tokenizer.getToken();
-        // Shorten "This is too long text" to "This is..."
-        const MAX_LENGTH: number = 40;
-        let problemText: string = token.text.trim();
-        if (problemText.length > MAX_LENGTH) {
-          problemText = problemText.substr(0, MAX_LENGTH - 3).trim() + '...';
+
+        if (token.text.trim().length) {
```

Comment:
>token.text.trim().length [](start = 12, length = 24)

Can you make this an explicit `!== 0` condition?

Reply:
Fixed in PR #370